go-controller/pkg/ovn/zone_interconnect/zone_ic_handler.go

```diff
@@ -568,6 +569,7 @@ func (zic *ZoneInterconnectHandler) cleanupNodeTransitSwitchPort(nodeName string
 // ip4.dst == 10.244.0.0/24 , nexthop = 100.88.0.2
 // ip4.dst == 100.64.0.2/16 , nexthop = 100.88.0.2  (only for default primary network)
 func (zic *ZoneInterconnectHandler) addRemoteNodeStaticRoutes(node *corev1.Node, nodeTransitSwitchPortIPs, nodeSubnets, nodeGRPIPs []*net.IPNet) error {
+	ops := make([]ovsdb.Operation, 0, 2)
```

Comment:
NIT: the capacity here won't be correct in dual stack, it doesn't matter much though.

Reply:
ok if i push another revision i'll bump it to 4